Description: Change the generation of the part-of-speech lexicon to use nstore instead of store.
 This should make the package architecture-independent.
Forwarded: no
Author: Chris Butler <chrisb@debian.org>
Last-Update: 2010-06-03

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -29,7 +29,7 @@
         
 
 sub install {
-        use Storable;
+        use Storable qw/nstore/;
         use File::Spec;
         my $lex_dir = 'Tagger';
         my $word_path = File::Spec->catfile( $lex_dir, 'pos_words.hash' );
@@ -40,8 +40,8 @@
                 _load_tags( File::Spec->catfile( $lex_dir, 'tags.yml' ) );
                 _load_words( File::Spec->catfile( $lex_dir, 'words.yml' ) );
                 _load_words( File::Spec->catfile( $lex_dir, 'unknown.yml' ) );
-                store \%_LEXICON, $word_path;
-                store \%_HMM, $tag_path;
+                nstore \%_LEXICON, $word_path;
+                nstore \%_HMM, $tag_path;
         }
 
         if( -f $word_path and -f $tag_path ){
